#!/usr/bin/env python
PACKAGE = "gate_detector"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("hue_lower", int_t, 0, "Hue Lower", 28 , 0, 255)
gen.add("hue_upper", int_t, 0, "Hue Upper", 54 , 0, 255)
gen.add("sat_lower", int_t, 0, "Sat Lower", 136, 0, 255)
gen.add("sat_upper", int_t, 0, "Sat Upper", 255, 0, 255)
gen.add("val_lower", int_t, 0, "Val Lower", 111, 0, 255)
gen.add("val_upper", int_t, 0, "Val Upper", 255, 0, 255)

exit(gen.generate(PACKAGE, "gate_detector", "Detector"))
